webapp/src/components/form/FormFileUpload.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from "react";
+import FormGroup from "./FormGroup";
+import FormToolTip from "./FormToolTip";
+import "./Style.css";
+
+// uploading, uploadPercentComplete, uploadFile, uploaded
```

Comment:
Think you accidentally included a comment :)

Reply:
Oh right - thanks :)